src/chapters/errors-and-debugging/debugging-logic-errors.rst

```diff
@@ -0,0 +1,154 @@
+========================
+Debugging Logic Errors
+========================
+
+We can debug runtime and syntax errors using the error messages produced.  
+Logic errors, however, do not always produce error messages. This sometimes makes them tougher to debug.
+
+While we can't provide a step-by-step approach that applies to every possible logic error, 
+we *can* give you some solid strategies. Two such strategies---using debugger tools and writing tests---will 
+be covered in future lessons. In this section, we start with a basic and effective way to debug logic errors.
+
+Printing Values
+---------------
+This example will work with logic and runtime errors.  These are commonly seen in the C# environment.
+
+When your code runs but doesn't produce the expected results, it is important to check the values of the 
+variables being used.
+
+Let's look at a program that has a logical bug.  
+
+This program is designed to take ask the user for the temperature in celsius and then convert it into Kelvin.  
+
+Before we get into the code let's think about all the parts: Get a number from the user then use it in a math equation.
+
+Okay, let's get building:
+
+.. replit:: csharp
+   :linenos:
+   :slug: Degrees-C-to-K-Logic-Error-CSharp
+   
+      Console.WriteLine("Temp in degrees C:");    
+      string degreesC = Console.ReadLine();             
+      Console.WriteLine("Degrees K: " + degreesC + 273.15); 
```

Comment:
not a huge deal but this code block and at least the one that follows have an extra tab on the page when they get rendered:

<img width="1124" alt="Screen Shot 2021-03-22 at 9 17 09 AM" src="https://user-images.githubusercontent.com/20828430/112004341-b39a9000-8aef-11eb-9150-092d8a4c3a60.png">